### PR TITLE
UI/UX refactor settings page

### PR DESCRIPTION
**Pull request by mhizterkeyz**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

**Comment by kofimokome** [comment]
Please send a screenshot of what you have done

**Comment by kofimokome** [comment]
@kadetXx review UI. Please

File: resources/views/backend/settings/settings.blade.php [hunk 1]
```diff
@@ -33,33 +50,163 @@
...
+                                    <div class="col-9 pt-1" style="margin-left: -25px;padding-left: 0;">
+                                        <p class="mb-1">
+                                            <b>
+                                                {{ isset($user_details['first_name'])  &&  isset($user_details['last_name']) ? $user_details['first_name'] ." ". $user_details['last_name']: "please update your name" }}
+                                            </b>
+                                        </p>
+                                        <p class="my-0">
+                                            {{ isset($user_details['email']) ? $user_details['email'] : "plese update your email"}}
+                                        </p>
+                                        @php if (isset($user_details['is_active'])): @endphp
```
Comment (kofimokome) [inline resources/views/backend/settings/settings.blade.php:77]: you can use `@if @else @endif` in place of `@php if() else @endphp`

File: resources/views/backend/settings/settings.blade.php [hunk 2]
```diff
@@ -164,7 +311,7 @@ function time_relative_to_time($ts) {
 
                                 <div class="tab-pane fade" id="user-profile">
                                     <div class="tile user-settings">
-                                        <h4 class="line-head">Profile</h4>
```
Comment (Douglasokolaa) [inline resources/views/backend/settings/settings.blade.php:167]: Edit profile, will be better here

File: resources/views/backend/settings/settings.blade.php [hunk 3]
```diff
@@ -164,7 +311,7 @@ function time_relative_to_time($ts) {
...
+                                        <h4 class="line-head">Edit Data</h4>
```
Comment (Douglasokolaa) [inline resources/views/backend/settings/settings.blade.php:314]: Edit profile will be better here

**Comment by Douglasokolaa** [comment]
Did you test the change password?

**Pull request merged**